recipes/atk/all/conanfile.py

```diff
@@ -49,24 +50,30 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("glib/2.76.3")
+        self.requires("glib/2.77.1")
 
     def validate(self):
         if self.options.shared and not self.dependencies["glib"].options.shared:
             raise ConanInvalidConfiguration(
                 "Linking a shared library against static glib can cause unexpected behaviour."
             )
 
-        if str(self.settings.compiler) == "Visual Studio" and not self.options.shared and \
-           is_msvc_static_runtime(self) and self.dependencies["glib"].options.shared:
-            raise ConanInvalidConfiguration("this specific configuration is prevented due to internal c3i limitations")
+        if (
```

Comment:
No please read the error message and compare to ci configurations currenty tested. There is no reason to extend this invalid configuration to compiler=msvc

Reply:
That error message does not really provide any hint or context for this being strictly related to VS and not msvc. Could you elaborate a bit, please?

Reply:
It says its related to c3i issues, it's a v1 pipeline workaround, not a real compiler issue, that's all and you know that v2 pipeline doesn't test this configuration so there is no reason for this change.